INSTALL.rst

```diff
@@ -152,6 +257,15 @@ You can now load the demo records (licenses, funders, grants, records):
    (zenodo)$ zenodo migration reindex -t recid
    (zenodo)$ zenodo index run -d
 
+Finally, run the Zenodo application:
+
+.. code-block:: console
+
+    (zenodo)$ export FLASK_DEBUG=True
+    (zenodo)$ zenodo run
```

Comment:
Maybe mention the `--reload` flag for live reloads. Also sometimes running `--with-threads` helps with pages that do many background requests (eg. search)